native_store.go

```diff
@@ -30,6 +31,7 @@ const (
 	remoteCredentialsPrefix       = "docker-credential-"
 	emptyUsername                 = "<token>"
 	errCredentialsNotFoundMessage = "credentials not found in native keychain"
+	errExeNotInPathMessage        = "exec: \"docker-credential-desktop.exe\": executable file not found in $PATH"
```

Comment:
Should there be a native_store_wsl.go to handle this that or can we count on a generic match?

Reply:
We don't think a separate `wsl.go` file is necessary. We have changed exact string matching to error matching to improve the code.